CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Fixed
 - `account delete` command: It is no longer necessary to provide the `--url` argument each time. Either the `--url` or `--network` argument must be provided, but not both, as they are mutually exclusive.
 
+### Forge
+
+#### Changed
+
+- Test performance has been optimized - using filter and `--exact` flag will result in compilation of only the selected test.
```

Comment:
```suggestion
- When using test name filter with `--exact` flag, forge will try to compile only the selected test.
```

It's not 100% true that it will only compile this tests. In case there are multiple matches by name, we'll compile all of them before filtering.

Reply:
Correct, it's because out false-positive approach, thanks for catching